tests/test_in/itest_lin_report_failures.rst

```diff
@@ -14,9 +14,9 @@ Integration test report for itest_lin_report_failures
 
     Test result: Fail
 
-    Directory 'C:\nonexistent' does not exist.
+    AssertionError: Directory 'C:\nonexistent' does not exist.
 
-    Second failure
+    Error: Second failure
```

Comment:
Kinda strange feeling. It is description of report, level is error, so kinda funy that we re-mention it. Makes sense for assert errors or type errors, but otherwise its just a plain failure...